pyproject.toml

```
@@ -1,10 +1,6 @@
 [build-system]
-requires = [
-    "setuptools >= 59.0",
-    "setuptools_scm",
-    "wheel",
-]
-build-backend = "setuptools.build_meta"
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
```

nit: alphabetize (also good to place lists as the last items
```suggestion
build-backend = "scikit_build_core.build"
requires = ["scikit-build-core"]
```

---

sure, make sense